src/components/controllers/tutorial/index.jsx

```diff
@@ -134,15 +128,14 @@ export class Tutorial extends Component {
 		this.setState({ error: null });
 	};
 
-	render({ html, meta, loading }, { code, error }) {
+	render({ html, meta, loading: routerLoading }, { code, error, loading: tutorialLoading }) {
```

Comment:
Drive-by loading mechanism improvement over #1143 

Still needs work, but incorporates both routing (markdown and tutorial bundles) with the tutorial orchestration (editor & repl runner) better.